tests/Mails/MailJet/SandboxHelperTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Mails\MailJet;
+
+use App\Domain\Mail\Addresses;
+use App\Mails\MailJet\SandboxHelper;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Mime\Address;
+use PHPUnit\Framework\Attributes\DataProvider;
+
+final class SandboxHelperTest extends TestCase
+{
+    #[DataProvider('sandboxModeProvider')]
+    public function test_get_sandbox_mode(bool $sandboxMode, array $allowedDomains, array $addresses, bool $expected): void
+    {
+        $sandboxHelper = new SandboxHelper($sandboxMode, $allowedDomains);
+        $to = new Addresses(array_map(static fn ($address) => new Address($address), $addresses));
+
+        $this->assertSame($expected, $sandboxHelper->getSandboxMode($to));
+    }
+
+    public static function sandboxModeProvider(): array
+    {
+        $allowedDomains = ['publiq.be', 'smurfen@gmail.com'];
+
+        return [
+            'Sandbox mode is off, should return false' => [false, $allowedDomains, ['test@gmail.com'], false],
```

Comment:
This is rather hard to read when everything is packed together on one line.

Reply:
I think you also missed a test for when the sandbox mode is off, but the email matches the allowed domains. I know the internal implementation and I know this doesn't make a difference, but it's a valid case we need to prevent side effects from a refactor.